mysite/settings/base.py

```diff
@@ -427,5 +425,12 @@ def root(*path):
         fg="red"
     ))
 
-if len(sys.argv) > 1 and sys.argv[1] in ['test']:
+def _is_running_tests():
+    if len(sys.argv) > 1 and sys.argv[1] in ['test']:
+        return True
+    if os.environ.get('RUN_ENV') == 'test':
+        return True
+    return False
+
+if _is_running_tests:
```

Comment:
This should be `_is_running_tests()`. What you've written here is valid syntax, but its doing something like

```python
if <function _is_running_tests at 0x7fda770625f0>:
   from .testing import *  # noqa
```

rather than evaluating the function, which is what you intended.